utils/automation/FoScanner/FormatResults.py

```diff
@@ -0,0 +1,114 @@
+#!/usr/bin/env python3
+
+# Copyright (C) 2024 Rajul Jha <rajuljha49@gmail.com>
+
+# SPDX-License-Identifier: GPL-2.0-only
+
+import re
+import os
+import binascii
+
+from .CliOptions import CliOptions
+
+class FormatResult:
+    """
+    For formatting the results from scanners with line number information
+    """
+
+    def __init__(self,cli_options:CliOptions):
+        self.cli_options = cli_options
```

Comment:
As a best practice, please define class variable `cli_options` in the class body as well.

Reply:
Okay! I've corrected it.